client/extensions/woocommerce/state/sites/review-replies/test/selectors.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getReviewReplies,
+	getReviewReply,
+} from '../selectors';
+import reviewReplies from './fixtures/review-replies';
+
+const preInitializedState = {
+	extensions: {
+		woocommerce: {},
+	},
+};
+const loadingState = {
+	extensions: {
+		woocommerce: {},
+	},
+};
+const loadedState = {
+	extensions: {
+		woocommerce: {
+			sites: {
+				123: {
+					reviewReplies: {
+						555: reviewReplies,
+					}
+				},
+				321: {
+					reviewReplies: {
+						556: [],
+					}
+				},
+			},
+		},
+	},
+};
+
+const loadedStateWithUi = { ...loadedState, ui: { selectedSiteId: 123 } };
+
+describe( 'selectors', () => {
+	describe( '#getReviewReplies', () => {
+		it( 'should be false when woocommerce state is not available.', () => {
+			expect( getReviewReplies( preInitializedState, 555, 123 ) ).to.be.false;
+		} );
+
+		it( 'should be false when review replies are loading.', () => {
+			expect( getReviewReplies( loadingState, 555, 123 ) ).to.be.empty;
+		} );
+
+		it( 'should be the list of replies if they are loaded.', () => {
+			expect( getReviewReplies( loadedState, 555, 123 ) ).to.eql( reviewReplies );
+		} );
+
+		it( 'should be the list of replies if they are loaded.', () => {
```

Comment:
Maybe a different string for this test to disambiguate from site 123's same-named test